src/modules/data/modelData.c

```diff
@@ -243,6 +244,20 @@ void lovrModelDataCopyAttribute(ModelData* data, ModelAttribute* attribute, char
     } else {
       lovrUnreachable();
     }
+  } else if (type == SN10x3) {
+    if (attribute->type == F32) {
+      for (uint32_t i = 0; i < count; i++, src += attribute->stride, dst += stride) {
+        float x = ((float*) src)[0];
+        float y = ((float*) src)[1];
+        float z = ((float*) src)[2];
+        float w = attribute->components == 4 ? ((float*) src)[3] : 0.f;
+        *(uint32_t*) dst =
+          ((((uint32_t) (int32_t) (x * 511.f)) & 0x3ff) <<  0) |
+          ((((uint32_t) (int32_t) (y * 511.f)) & 0x3ff) << 10) |
+          ((((uint32_t) (int32_t) (z * 511.f)) & 0x3ff) << 20) |
+          ((((uint32_t) (int32_t) (w * 2.f)) & 0x003) << 30);
+      }
```

Comment:
Should add `else { lovrUnreachable(); }`.  Note that glTF requires normal vectors to be f32x3.